### PR TITLE
test in winner announcement in details page

### PR DESCRIPTION
**Pull request by Theepanguna**
I did a small test on the winner announcement on the details page.  is the logic okay or should I do it differently, I have pushed to the new branch feature/winner_announcement.

### CODE REVIEW

File: src/app/views/tournament/tournament.component.ts [hunk 1]
```diff
@@ -21,10 +24,38 @@ export class TournamentComponent implements OnInit{
...
 
   get isAuthenticated() {
     return this.authService.isAuthenticated;
   }
+
+  onTabChange(event: MatTabChangeEvent) {
+    if (event.index === 4) {
+      this.fireConfetti();
+    }
+  }
```
Comment (SebastianSilvaDev) [inline src/app/views/tournament/tournament.component.ts]: This doesn't makes sense, you aren't waiting the person to show the winners tab, you are waiting that a player is marked as winner in the last round, then, you can change to the winner tab or use a dialog

**Review by SebastianSilvaDev — CHANGES_REQUESTED** [review]
The logic is a little bit faulty, it doesnt make sense to simple change to the last tab and see the winner, also, you arent accounting for the tournaments of 3 rounds

**Review by SebastianSilvaDev — COMMENTED** [review]
This looks good, do you need anything else so it can be approved?

**Comment by Theepanguna** [comment]
Yes, is it possible to have one test data from the Backend for rounds 2 and round 3. 
I tried to add a player but I guess still, we have not completed that task. 
(Because I think sometimes I need to check whether the round is completed or not.)

if there is any improvement needed on this I can work on it.
> This looks good, do you need anything else so it can be approved?

**Comment by Theepanguna** [comment]
Tested winner dialog is pushed to the branch for final approval

**Review by SebastianSilvaDev — APPROVED** [review]
LGTM

**Pull request merged**